src/parameterStore.ts

```diff
@@ -39,3 +38,16 @@ export async function getIntegrationTransformUrl(): Promise<string> {
 
     return data.Parameter!.Value!;
 }
+
+export default async function getIntegrationTransformData(): Promise<{
+    integrationTransformUrl: string;
+    integrationTransformAwsRegion: string;
+}> {
+    const promises = [
+        getParam('INTEGRATION_TRANSFORM_PATH', 'url'),
+        getParam('INTEGRATION_TRANSFORM_AWS_REGION_PATH', 'awsRegion'),
+    ];
```

Comment:
We should retrieve all the parameters in a single call. use the get-parameters API instead

Reply:
updated